static/app/views/replays/list/replaySearchAlert.spec.tsx

```diff
@@ -0,0 +1,120 @@
+import {render} from 'sentry-test/reactTestingLibrary';
+
+import {UpdateSdkSuggestion} from 'sentry/types';
+import {useLocation} from 'sentry/utils/useLocation';
+import usePageFilters from 'sentry/utils/usePageFilters';
+import useProjects from 'sentry/utils/useProjects';
+import {ReplaySearchMinSdkAlert} from 'sentry/views/replays/list/replaySearchAlert';
+
+jest.mock('sentry/utils/useProjects');
+jest.mock('sentry/utils/useLocation');
+jest.mock('sentry/utils/usePageFilters');
+
+const mockUseProjects = useProjects as jest.MockedFunction<typeof useProjects>;
+const mockUsePageFilters = usePageFilters as jest.MockedFunction<typeof usePageFilters>;
+const mockUseLocation = useLocation as jest.MockedFunction<typeof useLocation>;
+
+const project = TestStubs.Project();
+
+function getMockContext() {
+  return TestStubs.routerContext([{}]);
+}
+
+describe('ReplaySearchAlert', () => {
+  beforeEach(() => {
+    // @ts-ignore
```

Comment:
these `ts-ignore` statements kinda make the `jest.MockedFunction` part redundant